frontend/tests/acceptance/pages/signupPage.js

```diff
@@ -1,19 +1,59 @@
 const { I } = inject();
 
 module.exports = {
-  url: '/signup',
+  url: "/signup",
   fields: {
-    name: '//label[contains(text(),"name")]/parent::div/div/input[contains(@class, "MuiInputBase-input")]',
-    email: '//label[contains(text(),"email")]/parent::div/div/input[contains(@class, "MuiInputBase-input")]',
-    password: '//label[contains(text(),"password")]/parent::div/div/input[contains(@class, "MuiInputBase-input")]',
-    confirmPassword: '//label[contains(text(),"confirm password")]/parent::div/div/input[contains(@class, "MuiInputBase-input")]',
+    name: '//label[contains(text(),"Name")]/parent::div//input[contains(@class, "MuiInputBase-input")]',
+    email: '//label[contains(text(),"Email")]/parent::div//input[contains(@class, "MuiInputBase-input")]',
+    password: '//label[contains(text(),"Password")]/parent::div//input[contains(@class, "MuiInputBase-input")]',
+    confirmPassword: '//label[contains(text(),"Confirm password")]/parent::div//input[contains(@class, "MuiInputBase-input")]',
   },
-  signUp(name, email, password, confirmPassword) {
-    I.waitForVisible(this.fields.name);
+  elements: {
+    error_lbl: '//form/p[@class="makeStyles-error-5"]',
+    login_lbl: '//form/p/span[contains(@class,"makeStyles-span-7")]',
+    signup_btn: '//form/button/span[contains(.,"Sign Up")]',
```

Comment:
use better variable name. `lbl` and `btn` might be difficult to understand for many people.